treeherder/model/sql/template_schema/project_jobs_1.sql.tmpl

```diff
@@ -302,12 +302,11 @@ DROP TABLE IF EXISTS `series_signature`;
 CREATE TABLE `series_signature` (
   `signature` varchar(50) COLLATE utf8_bin NOT NULL,
   `property` varchar(50) COLLATE utf8_bin NOT NULL,
-  `value` varchar(150) COLLATE utf8_bin NOT NULL,
+  `value` text COLLATE utf8_bin NOT NULL,
   `active_status` enum('active','onhold','deleted') COLLATE utf8_bin DEFAULT 'active',
   KEY `idx_signature` (`signature`),
   KEY `idx_property` (`property`),
-  KEY `idx_value` (`value`),
```

Comment:
I can't currently think of a case where we would want to do that, but it's not inconceivable. It was a fairly arbitrary choice on my part to remove the index, maybe I should put it back.